### PR TITLE
http_content: add better example + packer-sdc: show sdk version

### PR DESCRIPTION
**Pull request by azr**

### CODE REVIEW

File: cmd/packer-sdc/main.go [hunk 1]
```diff
@@ -19,7 +20,7 @@ var (
 )
 
 func main() {
-	c := cli.NewCLI(app, "1.0.0")
+	c := cli.NewCLI(app, version.SDKVersion.String())
```
Comment (sylviamoss) [inline cmd/packer-sdc/main.go:23]: good catch!

File: cmd/packer-sdc/internal/renderdocs/docs-partials/packer-plugin-sdk/multistep/commonsteps/HTTPConfig-not-required.mdx [hunk 2]
```diff
@@ -7,14 +7,19 @@
...
   address and port of the HTTP server will be available as variables in
-  `boot_command`. This is covered in more detail below. Example: Setting
-  `"foo/bar"="baz"`, will allow you to http get on
-  `http://{http_ip}:{http_port}/foo/bar`.
+  `boot_command`. This is covered in more detail below.
+  Example:
+   ```hcl
+  .  http_content = {
+     "/a/b" = "content
+   }
```
Comment (sylviamoss) [inline cmd/packer-sdc/internal/renderdocs/docs-partials/packer-plugin-sdk/multistep/commonsteps/HTTPConfig-not-required.mdx:21]: I'd like an example more close to the reality, as the content will most of the time be in a file itself. 
```suggestion
  .  http_content = {
     "/a/b" = file("http/b")
   }
```

If you'd like to keep it that way - it's missing the last `"`
```suggestion
  .  http_content = {
     "/a/b" = "content"
   }
```

**Review by sylviamoss — APPROVED** [review]
Nice! 👍🏼  lgmt!

File: cmd/packer-sdc/internal/renderdocs/docs-partials/packer-plugin-sdk/multistep/commonsteps/HTTPConfig-not-required.mdx (hunk 2 above)
Comment (azr) [reply cmd/packer-sdc/internal/renderdocs/docs-partials/packer-plugin-sdk/multistep/commonsteps/HTTPConfig-not-required.mdx]: Ah good idea, I like the first example, also adding a templatefile example.

File: cmd/packer-sdc/internal/renderdocs/docs-partials/packer-plugin-sdk/multistep/commonsteps/HTTPConfig-not-required.mdx [hunk 3]
```diff
@@ -7,14 +7,20 @@
...
   address and port of the HTTP server will be available as variables in
-  `boot_command`. This is covered in more detail below. Example: Setting
-  `"foo/bar"="baz"`, will allow you to http get on
-  `http://{http_ip}:{http_port}/foo/bar`.
+  `boot_command`. This is covered in more detail below.
+  Example:
+  ```hcl
+    http_content = {
+      "/a/b"     = file("http/b")
+      "/foo/bar" = templatefile("${path.root}/preseed.cfg", { packages = ["nginx"] })
```
Comment (sylviamoss) [inline cmd/packer-sdc/internal/renderdocs/docs-partials/packer-plugin-sdk/multistep/commonsteps/HTTPConfig-not-required.mdx:21]: love it!

**Pull request merged**